frontend/src/components/ApplicantInformation/index.tsx

```diff
@@ -0,0 +1,326 @@
+          </Column>
+        </Row>
+        <Row className="seedlot-species-dropdown">
+          <Column sm={4} md={4} lg={10}>
+            <Dropdown
+              id="seedlot-species-dropdown"
+              ref={speciesInputRef}
+              titleText="Seedlot species"
+              helperText="Type or search for the seedlot species using the drop-down list"
+              label="Enter or choose an species for the seedlot"
+              invalid={invalidSpecies}
+              invalidText="Please select a species"
+              items={geneticClasses}
+              onChange={(e: any) => inputChangeHandlerSpecies(e)}
+            />
+          </Column>
+        </Row>
+        <Row className="class-source-radio">
+          <Column sm={4} md={8} lg={16}>
+            <RadioButtonGroup
+              legendText="Class A source"
+              name="class-source-radiogroup"
+              orientation="vertical"
+              defaultSelected="tested"
+              onChange={(e: string) => inputChangeHandlerRadio(e)}
+            >
+              <RadioButton
+                id="tested-radio"
+                labelText="Tested parent trees"
+                value="tested"
+              />
+              <RadioButton
+                id="untested-radio"
+                labelText="Untested parent trees"
+                value="untested"
+              />
+              <RadioButton
+                id="custom-radio"
+                labelText="Custom seedlot"
+                value="custom"
+              />
+            </RadioButtonGroup>
+          </Column>
+        </Row>
+        <Row className="registered-checkbox">
+          <Column sm={4} md={8} lg={16}>
+            <label htmlFor="registered-tree-seed-center" className="bcgov--label">
+              To be registered?
+            </label>
+            <Checkbox
+              id="registered-tree-seed-center"
+              name="registered"
+              labelText="Yes, to be registered with the Tree Seed Centre"
+              defaultChecked
+              onChange={(e: React.ChangeEvent<HTMLInputElement>) => inputChangeHandlerCheckboxes(e)}
+            />
+          </Column>
+        </Row>
+        <Row className="collected-checkbox">
+          <Column sm={4} md={8} lg={16}>
+            <label htmlFor="collected-bc" className="bcgov--label">
+              Collected from B.C. source?
+            </label>
+            <Checkbox
+              id="collected-bc"
+              name="collectedBC"
+              labelText="Yes, collected from a location within B.C."
+              defaultChecked
+              onChange={(e: React.ChangeEvent<HTMLInputElement>) => inputChangeHandlerCheckboxes(e)}
+            />
+          </Column>
+        </Row>
+        <Row className="save-button">
+          <Column lg={8}>
+            <Button renderIcon={DocumentAdd} type="submit">
+              Save and new create seedlot
```

Comment:
I belive here should be "Save and create new seedlot". But I'm not confident about this UX Writing, since new and create are the samething.

Reply:
Save and create new seedlot sounds correctly, I've adjusted :)